### PR TITLE
Ap delete create comments

### PR DESCRIPTION
**Pull request by atphy**
#### Changes Made
1. Modified `PostDetail.js` to remove link to unused new comment page
2. Modified appview to remove reference to new comment page, as add and view comments are on the same page in wireframe
3. Modified `Comment.js` to include delete functionality
4. Modified `CommentList.js` to include create functionality and refresh on add or delete

#### Steps to Review
1. Checkout this branch locally and run the application.
```
git fetch --all
git checkout ap-delete-create-comments
```
2. Run the application.
```
npm start
```
3. Initialize virtual environment, and run the server.
```
pipenv shell
python manage.py server
```
4. Test app functionality.
> Navigate to http://localhost:3000/
> Click on a post title
> Click on "View or Add Comments" button
> Add a comment, then click save
> Delete any comment
NOTE: Form does not currently clear after submit; will create new bug fix ticket since basic functionality is present

5. View code file.
> Confirm file modifications are present as indicated above.
> Confirm no unused code or extraneous comments exist.

### CODE REVIEW

**Review by michaelc321 — APPROVED** [review]
👍

File: src/components/comments/CommentForm.js [hunk 1]
```diff
@@ -15,17 +15,16 @@ export const CommentForm = (props) => {
...
                 subject: comment.subject,
                 content: comment.content,
                 post_id,
                 user_id: parseInt(localStorage.getItem("rare_user_id")),
-                timestamp: Date.now()
             }
             addComment(newCommentObject)
-                .then(props.history.push(`/comments/${post_id}`))
+                .then(props.getCommentsForPost)
         }else{
```
Comment (Raparker88) [inline src/components/comments/CommentForm.js:28]: I think here you just need to add
```
.then(()=> {
                const newComment = {}
                setComment(newComment)
                document.getElementById("commentForm").reset()
            })

Comment (atphy) [reply src/components/comments/CommentForm.js:28]: This clears out the subject but not the content for me--am I dropping it in there incorrectly?

**Review by heymonicakay — APPROVED** [review]
BEAUTIFUL!!

**Pull request merged**